src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/PrivilegesInterceptorImpl.java

```diff
@@ -203,6 +224,7 @@ private void replaceIndex(final ActionRequest request, final String oldIndexName
                 }
 
                 if (ar instanceof IndexRequest) {
+                    createIndexRequest = checkAndCreateRequestIfAbsent(newIndexName);
```

Comment:
Why are these requests done differently ? 

`createIndexRequest = checkAndCreateRequestIfAbsent(newIndexName);` 
and
`((CreateIndexRequest) request).index(newIndexName + "_1").alias(new Alias(newIndexName));`

Reply:
One creates new `CreateIndexRequest`, another modifies an existing `CreateIndexRequest`. For `CreateIndexRequest` we don't need to create another request.

Reply:
The code on line 227 and 265 are the same:
```
createIndexRequest = checkAndCreateRequestIfAbsent(newIndexName);
```

Reply:
I meant to ask about 216 but I see its a CreateIndexRequest